src/main/java/com/kobylynskyi/graphql/codegen/model/graphql/GraphQLRequests.java

```diff
@@ -10,8 +10,14 @@
 public class GraphQLRequests {
 
     private final List<GraphQLRequest> requests = new ArrayList<>();
+    private final String customOperationName;
```

Comment:
Please change the order of the fields to match the order of constructor parameters.